emission/storage/decorations/timeline.py

```diff
@@ -61,12 +63,12 @@ def get_aggregate_timeline_from_dt(start_dt, end_dt, box=None):
         logging.info("About to query for %s -> %s in %s" % (start_dt, end_dt, box))
     result_cursor = edb.get_timeseries_db().find({"data.local_dt": {"$gte": start_dt, "$lte": end_dt}}).sort("metadata.write_ts")
     logging.debug("result cursor has %d entries" % result_cursor.count())
-    result_list = list(result_cursor)
-    logging.debug("result list has %d entries" % len(result_list))
```

Comment:
You removed this line, but then we don't have an idea of how long it takes to access the cursor (e.g. to get the cursor count). Please restore a similar line in your next commit.